frontend/app/middleware/eventAccessGuard.global.ts

```diff
@@ -13,13 +13,12 @@ export default defineNuxtRouteMiddleware(async (to) => {
         switch (response.code) {
           case 403:
             description = 'Nie możesz otworzyć tej strony przed rozpocząciem wydarzenia'
-            break
+            return '/timer'
```

Comment:
It shouldn't work like that. You can set param here, redirect to error page with custom field. So if i try to open `/tasks` i get counter on the tasks page